staging_docs/user/guides/04-cache.md

```diff
@@ -0,0 +1,114 @@
+# Cache rubygems.org 
+
+In this guide you will configure Pulp to act as a pull-through cache of rubygems.org. After this,
```

Comment:
Please don't do the block formatting. We want to lower the barrier for contributions.

Reply:
I don't understand what that means. Can you please suggest a specific change?